src/Files.App/Data/Commands/ModifiableCommand.cs

```diff
@@ -78,7 +112,11 @@ public Task ExecuteAsync()
 				return BaseCommand.ExecuteAsync();
```

Comment:
```suggestion
				return BaseCommand.ExecuteAsync(parameter);
```
The same applies to L110.

Reply:
Done